react/components/ProductSummary.js

```diff
@@ -14,13 +14,19 @@ import { ProductContextProvider } from 'vtex.product-context'
 import { useCssHandles } from 'vtex.css-handles'
 
 import ProductSummaryContext from './ProductSummaryContext'
-import { productShape } from '../utils/propTypes'
+import { productShape, skuShape } from '../utils/propTypes'
 import { mapCatalogProductToProductSummary } from '../utils/normalize'
 
 const PRODUCT_SUMMARY_MAX_WIDTH = 300
 const CSS_HANDLES = ['container', 'containerNormal', 'element', 'clearLink']
 
-const ProductSummaryCustom = ({ product, actionOnClick, children, href }) => {
+const ProductSummaryCustom = ({
+  product,
+  actionOnClick,
+  children,
+  href,
+  selectedItem: currentSelectedItem,
```

Comment:
Why do you need to receive this from props instead of just getting from the context?

Reply:
because in the ProductSummaryContext the selectedItem would be the first sku by default, but in this case I want to send the `selectedItem` of the product page to the `ProductSummary`, as you can see in the two workspaces:
[in the first](https://thalyta1--storecomponents.myvtex.com/blouse-with-knot/p?skuId=310124164), without the changes, the product displayed in the `BuyTogether` is in the default sku color (grey) while [in the second workspace](https://selecteditem--storecomponents.myvtex.com/blouse-with-knot/p?skuId=310124164) it will have the same color that you select on the product page